### ISSUE TITLE
MMIs can't see Status tab to see shuttle launch.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#17223]
(WEB REPORT BY: jaredfogle REMOTE: 172.93.109.202:7777)
# Revision
c49cf5da3d473b2c5a34b0aa53a7b88139432435

# Description
I got blown as a borg, and as an MMI, I cannot see the Status tab and see when the shuttle launches.

# Steps to Reproduce
Be an MMI. Maybe it's not just MMIs, but blown borgs? Dunno.

# What you Expected
To see the Status tab

# What Actually Happened
I can't see it.

**Comment by jknpj** [comment]
You're just a brain

**Comment by Pathid** [comment]
Yeah, not everything can. Monkeys, nymphs and mice can't see the shuttle ETA either. But xenos, borers, and the deafest, blindest and naked-est humanoids somehow know how long it is until the shuttle at all times.

**Comment by DamianX** [comment]
It's a QoL thing. Should we just allow all mobs to see it?

**Comment by Dacendeth** [comment]
I think it would be fine if everyone just got to see it, everyone in space could just be really good at keeping track of time after they hear the shuttle call announcement.
